02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
+            <h6> Narcos </h6>
```

Comment:
Whitespace is stripped in HTML, so this works. But I would rather put block elements on their own line. Makes it easier to find errors in indentation and improves readability.